docs/src/IceNucleation.md

```diff
@@ -47,6 +47,25 @@ Both parameters are dependent on aerosol properties and temperature.
     freezing occurs in a different ice nucleation mode
     (either a second deposition or other condensation type mode).
 
+## Water Activity Based Deposition Nucleation
+The water activity based deposition nucleation model is analagous to ABIFM
+  for immersion freezing (see `ABIFM for Sulphuric Acid Containing Droplets`
```

Comment:
Maybe something like this would link it?:

[ABIFM for Sulphuric Acid Containing Droplets](https://clima.github.io/CloudMicrophysics.jl/dev/DevelopersGuide/#ABIFM for Sulphuric Acid Containing Droplets)